<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrescriptionPage</class>
 <widget class="QWidget" name="PrescriptionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1900</width>
    <height>850</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="infoLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>311</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Patient Information&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="prescriptionLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>301</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Prescription Details :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="prescriptionEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>180</y>
     <width>1791</width>
     <height>541</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Edit prescription informations here ....&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveBtn">
   <property name="geometry">
    <rect>
     <x>1639</x>
     <y>110</y>
     <width>211</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Send Prescription</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backBtn">
   <property name="geometry">
    <rect>
     <x>1700</x>
     <y>760</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;Patient Information :&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
